aten/src/ATen/native/quantized/cpu/fbgemm_utils.cpp

```diff
@@ -368,14 +368,14 @@ Tensor ConvertConvWeightsToChannelLastTensor<3>(
         "quantized", "Conv" + c10::to_string(kSpatialDim) + "dPackedParamsBase")
     .def_pickle(
         [](const c10::intrusive_ptr<ConvPackedParamsBase<kSpatialDim>>& params)
-        -> ConvParamsSerializationType { // __getstate__
```

Comment:
this was pretty nice to have when grepping around to be able to quickly see what is the return type, would be sad to see it go, but I understand if it's necessary

Reply:
Looks like this breaks in Visual C++ . :/